README.rst

```diff
@@ -84,6 +84,7 @@ Besides the numerical argument, there are two main optional arguments.
 * ``en_IN`` (English - India)
 * ``es`` (Spanish)
 * ``es_CO`` (Spanish - Colombia)
+* ``es_GT`` (Spanish - Guatemala)
```

Comment:
@monroy95 then we don't need this change.

Reply:
Added suggested change